### PR TITLE
refactor: removed message field from TrialLog [DET-8983]

### PR DESCRIPTION
**Pull request by corban-beaird**
## Description
Removed:
- Deprecated message field from TrialLog
- `Resolve` function used to resolve that message field
- Message field in TrialLogsResponse

## Test Plan
Ran existing testing suite

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
DET-8983

### CODE REVIEW

**Comment by corban-beaird** [comment]
It looks like there ere some additional e2e tests that I missed, getting those fixed up!

**Review by stoksc — COMMENTED** [review]
This field is still used by the CLI, so you will need to refactor those calls to print what the message used to contain based on the structured information it has

File: master/pkg/protoutils/protoconverter/converter.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func (c *ProtoConverter) ToStringList(x []uuid.UUID) []string {
 	return strUUIDList
 }
 
-// ToCheckpointState converts a proto chechkpoint state internal state represenations.
+// ToCheckpointState converts a proto checkkpoint state internal state represenations.
```
Comment (stoksc) [inline master/pkg/protoutils/protoconverter/converter.go:144]:
```suggestion
// ToCheckpointState converts a proto checkpoint state internal state represenations.
```

File: e2e_tests/tests/experiment/experiment.py [hunk 2]
```diff
@@ -404,7 +404,7 @@ def num_error_trials(experiment_id: int) -> int:
 
 def trial_logs(trial_id: int, follow: bool = False) -> List[str]:
     return [
-        tl.message
+        str(tl.log) if tl.log is not None else ""
```
Comment (stoksc) [inline e2e_tests/tests/experiment/experiment.py:407]: you fill need to fmt the old message, i think, for now

Comment (corban-beaird) [reply e2e_tests/tests/experiment/experiment.py]: Yep! I created a `format_trial_log` that follows the formatting that had been applied previously! I just got it pushed up 😄

File: harness/determined/common/api/logs.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 def pprint_task_logs(task_id: str, logs: Iterable[bindings.v1TaskLogsResponse]) -> None:
     try:
         for log in logs:
-            print(log.message, end="")
+            print(log.log, end="")
```
Comment (stoksc) [inline harness/determined/common/api/logs.py:12]: these should also print the formatted log, i think?

**Review by stoksc — COMMENTED** [review]
looking good. meta point, we should add a `--json` option for these.

**Comment by stoksc** [comment]
re: --json flag: doesn't need to be this pr.

**Comment by corban-beaird** [comment]
> re: --json flag: doesn't need to be this pr.

Sounds good! I'll make sure that the log messages are properly formatted everywhere 😄 Would this be another ticket?

**Pull request merged**